osu.Game/Overlays/SettingsSubPanel.cs

```diff
@@ -47,7 +47,9 @@ public BackButton()
             [BackgroundDependencyLoader]
             private void load()
             {
-                Size = new Vector2(SettingsSidebar.DEFAULT_WIDTH);
+                Size = new Vector2(SettingsSidebar.EXPANDED_WIDTH);
```

Comment:
Is it just my thinking that this button is _comically_ huge?

![1703619806](https://github.com/ppy/osu/assets/20418176/89b056e8-5b83-4a5a-835f-3cd0a42cfbbd)

I didn't change it myself because from the changes it *must* have been intentional, but it looks really too large to live to me.

Reply:
yeah it's a bit weird i guess? i'll scale it down.

Reply:
i did not expect the solution to this to be a huge padding but sure that works i guess